Content.Client/Popups/PopupSystem.cs

```diff
@@ -54,9 +54,15 @@ public void PopupEntity(string message, EntityUid uid)
             PopupMessage(message, _eyeManager.CoordinatesToScreen(transform.Coordinates), uid);
         }
 
-        public void PopupMessage(string message, ScreenCoordinates coordinates, EntityUid? entity = null)
+        private void PopupMessage(string message, ScreenCoordinates coordinates, EntityUid? entity = null)
         {
-            var label = new PopupLabel(_eyeManager, EntityManager)
+            var mapCoords = _eyeManager.ScreenToMap(coordinates);
+            PopupMessage(message, EntityCoordinates.FromMap(_map, mapCoords), entity);
+        }
+
+        private void PopupMessage(string message, EntityCoordinates coordinates, EntityUid? entity = null)
```

Comment:
why make these both private?

Reply:
Both? there's only a single one.

Why: this is an internal implementation method. Specifying both `EntityUid` and a coordinate pair never makes sense since the UID (if not null) will just override the coordinate pair always. it's written like this to share the code between the implementations but it shouldn't be a public API.